src/data/__snapshots__/redux.test.jsx.snap

```diff
@@ -333,12 +224,80 @@ Object {
 }
 `;
 
-exports[`Data layer integration tests Test resetExam Should reset exam, and update attempt and exam 1`] = `
+exports[`Data layer integration tests Test resetExam Should reset exam attempt 1`] = `
 Object {
   "examState": Object {
     "activeAttempt": null,
     "allowProctoringOptOut": false,
-    "apiErrorMsg": "Request failed with status code 404",
```

Comment:
example of a test that was basically just asserting it was throwing error...

Reply:
Are some of these test cases missing now? I agree that this perhaps wasn't the best approach but it looks like there are a few test cases missing for when the API errors now

Reply:
yeah I just dropped a few of these because the code  path for an api error is the same for most of these and we're just testing the same thing over and over and over.

That's probably a good test we could isolate and pull out into its own describe block. I'll add a comment in the code to mention this.